ckan/logic/action/create.py

```diff
@@ -225,15 +225,18 @@ def package_create(context, data_dict):
         model.repo.commit()
 
     # need to let rest api create
-    context["package"] = pkg
     # this is added so that the rest controller can make a new location
-    context["id"] = pkg.id
+    api_context = context.copy()
+    api_context["package"] = pkg
+    api_context["id"] = pkg.id
```

Comment:
let's just remove this instead, as you discussed

Reply:
👍